config/configauth/configauth.go

```diff
@@ -33,28 +33,28 @@ type Authentication struct {
 	AuthenticatorID component.ID `mapstructure:"authenticator"`
 }
 
-// GetServerAuthenticator attempts to select the appropriate ServerAuthenticator from the list of extensions,
+// GetServer attempts to select the appropriate Server from the list of extensions,
```

Comment:
Sorry for the confusion, I missed this in an earlier review: the current API should not change. The renaming should affect only the new types.

Reply:
Done